src/switchroot/ostree-prepare-root.c

```diff
@@ -570,9 +570,10 @@ main (int argc, char *argv[])
       else
         {
           /* Bind-mount /etc (at deploy path), and remount as writable. */
-          if (mount ("etc", TMP_SYSROOT "/etc", NULL, MS_BIND | MS_SILENT, NULL) < 0)
+          static const char *tmp_sysroot_etc = TMP_SYSROOT "/etc";
```

Comment:
There's another occurrence of TMP_SYSROOT "/etc" a few lines up if you want to catch that too.